app/views/facility_journals/new.html.haml

```diff
@@ -25,6 +25,8 @@
 = content_for :top_block do
   = render :partial => 'shared/transactions/top', :locals => { :tab => 'new_journal' }
 
+- if @problem_order_details.any?
```

Comment:
This might not be an issue, but it might be worth calling `to_a` before the call to `any?`. We're going to be iterating anyways. That would avoid an extra `count` query. I'd only worry about that if the query is slow.